README.md

```
@@ -529,10 +510,12 @@ Only bypass quarantine for an artifact you trust. The release workflow verifies
 
 ## Performance Measurement
 
+## Performance Measurement
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)